### PR TITLE
EXPANDR-7462 Updating code to check corner cases and re-organizing the code

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-7478
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-7462

## Description
Fixing issues reported by SOC.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by capanw** [comment]
@johnnywilkes @BigEasyJ  Please review when you guys can. Thanks

**Comment by shmuel44** [comment]
Hi @capanw,
Thank you for your contribution!
Good work :)

The code looks good to me. 
Are we still waiting for @johnnywilkes and @BigEasyJ?

**Comment by BigEasyJ** [comment]
@shmuel44 we are not required on this code change due to it not being a pack that is owned by our team.

**Comment by shmuel44** [comment]
Hi @capanw , the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR or you can record a video of the demo and send it 🙏

**Comment by capanw** [comment]
@shmuel44 Updating release notes version to avoid conflicts.

File: Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_13.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#### Scripts
+
+##### AWSRecreateSG
+- Updated the script to fix an issue with security group rule case.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_13.md:4]:
```suggestion
Updated the script to fix an issue with security group rule case.
```

**Comment by ShirleyDenkberg** [comment]
@shmuel44 Doc review completed.

**Review by shmuel44 — APPROVED** [review]
Good Job!

**Comment by shmuel44** [comment]
Hi @capanw,
I merged the internal PR, and your update is now in the master branch of content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Pull request merged**